packages/twenty-front/src/modules/object-record/record-field/form-types/components/SelectedVariableChip.tsx

```diff
@@ -0,0 +1,27 @@
+import { SortOrFilterChip } from '@/views/components/SortOrFilterChip';
+import { extractVariableLabel } from '@/workflow/search-variables/utils/extractVariableLabel';
+import styled from '@emotion/styled';
+
+export const StyledContainer = styled.div`
+  align-items: center;
+  display: flex;
+`;
+
+type SelectedVariableChipProps = {
+  rawVariable: string;
+  onRemove: () => void;
+};
+
+export const SelectedVariableChip = ({
```

Comment:
VariableChip as a name is enough I think. We have a RecordChip, that will be consistent!

Reply:
Great 👌

Reply:
Done